src/dynarec/rv64/dynarec_rv64_00_2.c

```diff
@@ -486,9 +486,10 @@ uintptr_t dynarec64_00_2(dynarec_rv64_t* dyn, uintptr_t addr, uintptr_t ip, int
             if (MODREG) { // reg <= reg? that's an invalid operation
                 DEFAULT;
             } else { // mem <= reg
-                addr = geted(dyn, addr, ninst, nextop, &ed, x2, x1, &fixedaddress, rex, NULL, 0, 0);
-                if (gd != ed) { MV(gd, ed); }
-                if (!rex.w || rex.is32bits) {
+                addr = geted(dyn, addr, ninst, nextop, &ed, gd, x1, &fixedaddress, rex, NULL, 0, 0);
+                if (gd != ed) {
+                    MVxw(gd, ed);
+                } else if (!rex.w && !rex.is32bits) {
```

Comment:
you are sure of this change? that sees odd to me.

Reply:
`rex.is32bits` goes to `geted_32`, where it is always zero-up'd.

Reply:
Ah ok, make sense then.